Cargo.toml

```diff
@@ -1,10 +1,7 @@
 [package]
 name = "cgl"
 license = "MIT / Apache-2.0"
-version = "0.3.2"
+version = "0.3.3"
```

Comment:
This needs to be 0.4.0, since the public api types are changing.

Reply:
Hmm, I shouldn't think so? `libc::c_int` is a type alias for `i32` (most of the time), and so is `std::os::raw::c_int` (again, most of the time, but the definitions should always match)

Reply:
I admit, I don't actually know what the rules are around type changes in the public api when the types involved are all aliases that resolve to the same underlying type. I guess we can try it.